README.md

```diff
@@ -33,9 +33,18 @@ command you've been using to start up the environment, but replace the `up` part
 0. Make sure the drive with the directory that will contain the sites is a shared drive (check Docker Desktop's
 Preferences > Resources > File Sharing)
 1. `cd authoring`
-2. Open the `docker-compose.yaml` in an editor and add the following volume to both the `tomcat` and the `deployer`
-service (assume `C` is the shared drive, and replace the `/host/path/to/sites` for the actual host path):
-`- c:/host/path/to/sites:/opt/crafter/data/repos/sites`
+2. Open the `docker-compose.yaml` in an editor and edit the ``crafter_data`` volume like the following (assume `C` is the shared drive, and replace the `/host/path/to/sites` for the actual host path):
```

Comment:
We need to keep @alhambrav 's edits that fix the performance issues. This change removes those edits and puts us back to where the docs weren't good enough. Please fix.

Reply:
Hi @sumerjabri 

Do you mean the change in this commit? https://github.com/craftercms/docker-compose/commit/212072c2cfadb886dd93004c958d82a1c95edf2e

What I did was to merge the newest `develop` to my feature branch (keep change in `develop`) as in here: https://github.com/craftercms/docker-compose/pull/26/commits/76ad5007ca3439d039f3bf2209c939f018284e22

In line 36, there is no change with the current `develop`.